shared/js/background/email-utils.es6.js

```diff
@@ -83,6 +86,52 @@ export const getAddresses = () => {
     }
 }
 
+function firePixel (pixelName, params) {
+    const randomNum = Math.ceil(Math.random() * 1e7)
+
+    const searchParams = new URLSearchParams(Object.entries(params))
+
+    const url = getURL(pixelName) + `?${randomNum}&${searchParams.toString()}`
+
+    console.log(url)
+
+    load.url(url)
+}
+
+function currentDate () {
+    return new Date().toLocaleString('en-CA', {
+        timeZone: 'America/New_York',
+        dateStyle: 'short'
+    })
+}
+
+const pixelName = (name, browserName) => {
+
+    return `${name}_${browserName.toLowerCase()}`
+}
+
+const addressUsed = (pixel) => {
+    const browserInfo = parseUserAgentString()
+    const browserName = browserInfo?.browser?.toLowerCase() ?? 'unknown'
+    if (browserName === 'firefox') return
+
+    const userData = getSetting('userData')
+    if (!userData?.userName) return
+
+    const lastAddressUsedAt = getSetting('lastAddressUsedAt') ?? ''
+
+    firePixel(pixelName(pixel, browserName), { duck_address_last_used: lastAddressUsedAt, cohort: userData.cohort })
+    updateSetting('lastAddressUsedAt', currentDate())
+}
+
+export const privateAddressUsed = () => {
+    addressUsed('email_filled_random_extension')
```

Comment:
Have we already settled on the name? We usually use "private" instead of "random" on user-facing UIs. Not sure about the pixels, though.

Reply:
This is for consistency with mobile. Mobile pixels are named `email_filled_{random,main}_{ios,android}_{phone,tablet}`

I also don't like that we use different vocabulary, but I'd rather keep the implementation consistent across platforms.